test/DrupalTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace EC\OpenEuropa\CodeReview\Test;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Collection\FilesCollection;
+use GrumPHP\Task\Context\GitPreCommitContext;
+
+/**
+ * Tests for Drupal conventions.
+ */
+class DrupalTest extends AbstractTest
+{
+    /**
+     * Tests different git messages against the predefined conventions.
+     *
+     * @param string $fixture
+     *   Name of the fixture.
+     * @param int    $expected
+     *   Expected result after the test.
+     *
+     * @dataProvider commitMessageProvider
+     */
+    public function testPhpCodeMessage($fixture, $expected)
+    {
+        $container = $this->getContainer($this->getDistPath().'/drupal-conventions.yml');
+        $collection = new FilesCollection([$this->getFixture($fixture)]);
+        $context = new GitPreCommitContext($collection);
+        /** @var \GrumPHP\Task\TaskInterface $task */
+        $task = $container->get('task.phpmd');
+        $result = $task->run($context);
+        $this->assertEquals($result->getResultCode(), $expected);
+    }
+
+    /**
+     * Test case provider function.
+     *
+     * Test file extensions.
+     *
+     * @return array
+     *      Test data.
+     */
+    public function commitMessageProvider()
+    {
+        return [
+          ['correct-code.theme', TaskResult::PASSED],
+          ['ignored-code.xxx', TaskResult::SKIPPED],
```

Comment:
@dxvargas could we add tests also for other file types?

Reply:
Done.